forcedotcom/commerce-on-lightning
# @W-13893187 feat : added shipping entries in store setup.

### What does this PR do?

Adds initialise shipping method to the setup script that populates the inital shipping entities for Native Shipping. 

### What issues does this PR fix or reference?
[W-13893187](https://gus.lightning.force.com/lightning/r/a07EE00001XiU4FYAV/view)

### How to Test/Testing Effort 

- Create and setup the store using commerce:store:create 
- Check for values under developer console
- Entities added : 
      ShippingConfigurationSet
      ShippingRateArea
      ShippingRateGroup
      StandardShippingRate
